### PR TITLE
Change how environment variables are checked

### PR DESCRIPTION
**Pull request by tooxie**
Environment variables are false if unset or set to empty string,
anything else is considered true.

### CODE REVIEW

File: aether-common-module/aether/common/conf/settings.py [hunk 1]
```diff
@@ -23,8 +23,10 @@
 # Common Configuration
 # ------------------------------------------------------------------------------
 
-DEBUG = (os.environ.get('DEBUG', '').lower() == 'true')
-TESTING = (os.environ.get('TESTING', '').lower() == 'true')
+# Environment variables are false if unset or set to empty string, anything
+# else is considered true.
+DEBUG = not (os.environ.get('DEBUG', '') == '')
```
Comment (obdulia-losantos) [inline aether-common-module/aether/common/conf/settings.py]: Don't you like the `!=` operator?

Comment (tooxie) [reply aether-common-module/aether/common/conf/settings.py]: Nah! That's too readable 😅

**Review by obdulia-losantos — DISMISSED** [review]

File: aether-common-module/aether/common/conf/settings.py [hunk 2]
```diff
@@ -23,8 +23,10 @@
 # Common Configuration
 # ------------------------------------------------------------------------------
 
-DEBUG = (os.environ.get('DEBUG', '').lower() == 'true')
-TESTING = (os.environ.get('TESTING', '').lower() == 'true')
+# Environment variables are false if unset or set to empty string, anything
+# else is considered true.
+DEBUG = os.environ.get('DEBUG', '') != ''
+TESTING = os.environ.get('TESTING', '') != ''
```
Comment (quesebifurcan) [inline aether-common-module/aether/common/conf/settings.py]: Alternative syntax:
```
DEBUG = True if os.environ.get('DEBUG') else False
```

**Review by quesebifurcan — DISMISSED** [review]

File: aether-common-module/aether/common/conf/settings.py (hunk 2 above)
Comment (tooxie) [reply aether-common-module/aether/common/conf/settings.py]: Taking it to the extreme:
```py
bool(os.environ.get('DEBUG'))
```

**Review by shawnsarwar — APPROVED** [review]

**Review by obdulia-losantos — APPROVED** [review]

**Review by quesebifurcan — APPROVED** [review]

**Pull request merged**